libs/datasets/region_aggregation.py

```diff
@@ -237,7 +237,7 @@ def _apply_scaling_factor(
     )
     # Check that scale_factors has location index and CommonFields in columns.
     assert scale_factors.index.names == [CommonFields.LOCATION_ID]
-    assert scale_factors.columns.difference(CommonFields).empty
+    assert scale_factors.columns.difference(CommonFields.list()).empty
```

Comment:
selecting columns with the CommonFields enum class raised:
```
raise TypeError("Input must be Index or array-like")
```

so I've created a class method to create a list of all the values